README.md

```diff
@@ -40,9 +40,7 @@ Options:
       --ignore-commit-pattern [regex] # pattern to ignore when parsing commits
       --tag-pattern [regex]           # override regex pattern for release tags
       --tag-prefix [prefix]           # prefix used in version tags, default: v
-      --starting-commit [hash]        # starting commit to use for changelog generation
```

Comment:
this is a semver major then?

Reply:
Yep. Will publish `2.0.0` once this is done and merged. It's not completely "breaking" as I guess it will just ignore this param and still generate a changelog (albiet one with more commits than desired) but a major bump feels cleaner.